packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md

```diff
@@ -1,36 +1,36 @@
-### Your app displays the sign-in page
+### 1: Your app displays the sign-in page
```

Comment:
Our style is to not number headings. Also something that is in the more in-depth style guide and not on the Git wiki